src/Evolution/Systems/Cce/Components/CharacteristicEvolution.hpp

```diff
@@ -98,6 +98,8 @@ template <class Metavariables>
 struct CharacteristicEvolution {
   using chare_type = Parallel::Algorithms::Singleton;
   using metavariables = Metavariables;
+  using cce_system =
+      Cce::System<Metavariables::uses_partially_flat_cartesian_coordinates>;
```

Comment:
The CCE system is defined correctly. How about replacing the CCE system in the PDAL with this type alias as well?